io/driver/module_unload_load.py.data/README.txt

```diff
@@ -1,13 +1,9 @@
-This Program loads and unloads the kernel driver modules. For the kernel driver modules that have dependencies, the dependant modules should  be listed in the config file (config). The script takescare of unloading the dependant modules, if any, before unloading the core module. It runs for a total of 100 times as stress.
-
-The script picks up the modules listed in lspci and performs the module unload/load operation.
-
-The test will PASS when at least one of the drivers finishes the whole loop. 
+This Program loads and unloads the kernel driver modules. For the kernel driver modules that have dependencies, the dependant modules should  be listed in the config file (config). The script takescare of unloading the dependant modules, if any, before unloading the core module. It runs for given number iterations.
 
 This script should be run as root.
 
 Inputs
 ------
-ITERATIONS -    No of counts to unload and load the module. Defaults to 10.
+ITERATIONS -    No of counts to unload and load the module. Defaults to 1.
```

Comment:
What is the reason for defaulting to 1 ?

Reply:
if it is for all the module then doing 10 times is really takes lot of time. hence made it to 1.
I think that should not matter here, as we have given a parameter to give the count.

Reply:
When changing existing values, it is always better to have reason documented. This discussion here is that documentation